packages/coinjoin/src/backend/getAccountUtxo.ts

```diff
@@ -7,7 +7,7 @@ type AddressPaths = {
 };
 
 const isCoinbaseUtxo = (tx: Transaction) =>
-    tx.details.vin.length === 1 && !!tx.details.vin[0].coinbase;
+    tx.details.vin.length === 1 && !tx.details.vin[0].isAddress && !tx.details.vin[0].txid;
```

Comment:
So if the tx has an input but the input is not an address and falsy txid then it is coinjoin tx?

This is the way blockbook give us tx right? Why not using `coinbase` attribute anymore? is it not reliable?

Reply:
Blockbook give us different tx data when asked `/tx` endpoint (or `getTransaction` ws method) and when asked `/block` endpoint (or `getBlock` ws method). Specifically, `coinbase` attribute is now missing as we get txs only from block and not one-by-one (as part of preserving privacy).

The condition means that the input is not an address and `vin.txid` (previous transaction for this input) is missing. I think the second part should be enough, if you want me to simplify it.